### ISSUE TITLE
PTerm UX for CLI (Make Atmos Prettier)

### ISSUE THREAD

**Issue by osterman** [cloudposse/atmos#367]
Be able to use atmos without understanding stack name conventions, and be prompted to select stack names.

```
atmos terraform plan vpc
```

Would provide a multi-selector to select that stacks.

**Comment by jamengual** [comment]
this will have to be `"enabled"` to keep the default behavior that is more cicd/pipeline friendly

**Comment by osterman** [comment]
@jamengual correct; we have to maintain friendly support for headless terminals / TTY-less terminals

**Comment by llpham14** [comment]
https://cloudposse.atlassian.net/browse/DEV-870

**Comment by osterman** [comment]
This was implemented:
- #679
